src/Fields/Fields.jl

```diff
@@ -256,9 +256,12 @@ local_geometry_field(space::AbstractSpace) =
 local_geometry_field(field::Field) = local_geometry_field(axes(field))
 
 
+get_dz(field::Field) = get_dz(axes(field))
+get_dz(space::AbstractSpace) = Field(Spaces.get_dz(space), space)
```

Comment:
Functios should be consistent in the object they return: `dz_data` returns a `DataLayout`, and `dz_field` returns a `Field`
```suggestion
dz_field(field::Field) = dz_field(axes(field))
dz_field(space::AbstractSpace) = Field(Spaces.dz_data(space), space)
```